src/instana/util/config.py

```diff
@@ -295,21 +300,35 @@ def get_disable_trace_configurations_from_env() -> Tuple[List[str], List[str]]:
     return [], []
 
 
+def get_tracing_root_key(config_data: Dict[str, Any]) -> Union[str, None]:
```

Comment:
It's a [best practice](https://docs.python.org/3/library/typing.html#typing.Optional) to use `Optional[X]` instead of `Union[X, None]`: 

```suggestion
def get_tracing_root_key(config_data: Dict[str, Any]) -> Optional[str]:
```